### PR TITLE
fix(featureDev): File Rejection stopped working

### PR DESCRIPTION
**Pull request by dianaabv16**
## Problem
when clicking on the X to reject change nothing happens

Fix was introduced to Mynah in 4.9.1 version.

On our side we needed to update mynah chat card update by messageId.

Tested:
Screenshot 2024-05-24 at 14 47 51

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by dogusata — APPROVED** [review]

**Review by sannicm — APPROVED** [review]

File: packages/core/src/amazonqFeatureDev/session/session.ts [hunk 1]
```diff
@@ -166,8 +166,13 @@ export class Session {
         return resp.interaction
     }
 
-    public async updateFilesPaths(tabID: string, filePaths: NewFileInfo[], deletedFiles: DeletedFileInfo[]) {
-        this.messenger.updateFileComponent(tabID, filePaths, deletedFiles)
+    public async updateFilesPaths(
+        tabID: string,
+        filePaths: NewFileInfo[],
+        deletedFiles: DeletedFileInfo[],
+        messageId: string
```
Comment (hayemaxi) [inline packages/core/src/amazonqFeatureDev/session/session.ts:173]: There's now 4 complex args for this function, and multiple other functions in this file use the same set of args.
To help with readability, consider passing a typed args object to each function with the same set of arguments instead, e.g.

```
type UpdateFileOpts {
  tabId: string;
  filePaths: NewFileInfo[];
  deletedFiles: DeletedFileInfo[];
  messageId: string;
}

public async updateFilesPaths(opts: UpdateFileOpts) {
   ...
}

updateFilesPaths({
  tabId: this.tabId,
  filePaths,
  deletedFiles,
  messageId: this.messageId
})
```

Comment (dianaabv16) [reply packages/core/src/amazonqFeatureDev/session/session.ts:173]: if its ok I will add it as a follow up.

**Comment by jpinkney-aws** [comment]
Can you add a changelog item for this

**Comment by dianaabv16** [comment]
Latest commit was to only update changelog.
Before that all checks were green

**Comment by justinmk3** [comment]
Rebasing on latest master will fix CI.

**Comment by justinmk3** [comment]
Ah, I see the problem. Unrelated to this PR

**Pull request merged**